Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -520,7 +520,15 @@ public final class PackageBuilder {
                 if subpath == "" || subpath == "." {
                     return packagePath
                 }
-                let path = packagePath.appending(RelativePath(subpath))
+
+                let path: AbsolutePath
+
+                if subpath.hasPrefix("/") {
+                    path = AbsolutePath(subpath)
+                } else {
+                    path = AbsolutePath(subpath, relativeTo: packagePath)
+                }
```

Comment:
After carefully going through `Path.swift`, I found this is redundant check.
`AbsolutePath` already has [`init(_ str: String, relativeTo basePath: AbsolutePath)`](https://github.com/apple/swift-package-manager/blob/95b41e9a45352dafb91fb264d31ff5024ba803aa/Sources/Basic/Path.swift#L62) which does this `if-else` check for us